google/cloud/bigtable/internal/partial_result_set_resume_test.cc

```diff
@@ -303,6 +303,9 @@ TEST(PartialResultSetResume, TooManyTransients) {
   auto status = reader->Finish();
   EXPECT_THAT(status,
               StatusIs(StatusCode::kUnavailable, HasSubstr("Try again")));
+  EXPECT_THAT(status.error_info().metadata(),
+              ::testing::Contains(::testing::Pair("gcloud-cpp.retry.reason",
+                                                  "retry-policy-exhausted")));
```

Comment:
not a blocker: Is `gcloud-cpp` by convention? Was curious since it could get confused with "gcloud cli".

Reply:
It's a convention we've used for some time. The ideal is that specifying `cpp` it should disambiguate from `cli`. This key/value pair came from https://github.com/googleapis/google-cloud-cpp/blob/4e642f7dcf13a05c4fd4a83aa261a4aad4a9fe76/google/cloud/internal/retry_loop_helpers.cc#L77